src/cmd/windows/durable_task_monitor.go

```diff
@@ -0,0 +1,150 @@
+// +build windows
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"os/signal"
+	"strings"
+	"sync"
+	"syscall"
+
+	"golang.org/x/sys/windows"
+	"jenkinsci.org/plugins/durabletask/common"
+)
+
+// Launches the script in a new session and waits for its completion.
+func launcher(wg *sync.WaitGroup, exitChan chan bool,
+	executable string, args string, resultPath string, outputPath string,
+	launchLogger *log.Logger, scriptLogger *log.Logger) {
+
+	defer wg.Done()
+	defer common.SignalFinished(exitChan)
+
+	var scriptCmd *exec.Cmd
+	var sysAttr windows.SysProcAttr
+
+	sysAttr.CreationFlags = windows.CREATE_NEW_PROCESS_GROUP
+	if executable == "cmd" {
+		scriptCmd = exec.Command(executable)
+		sysAttr.CmdLine = executable + " " + args
```

Comment:
Due to the way `cmd` parses strings (notably strings with spaces), `golang` does need to know when `cmd` is being called in order to use the `CmdLine` field.